.env_SAMPLE

```diff
@@ -33,15 +33,16 @@ export ALLOW_ADMIN_URL=True
 #export AKAMAI_HOST=<akamai_hostname>
 
 ############################
-# Mysql Database Server.
+# MySQL Database Server
 ############################
 
 export MYSQL_NAME=v1
+export MYSQL_HOST=localhost
 export MYSQL_USER=root
 #export MYSQL_PW=<password>
-#export MYSQL_HOST=<hostname>
 #export MYSQL_PORT=<portnumber>
 
+export STORAGE_ENGINE="SET default_storage_engine=MYISAM"
```

Comment:
This is another one of those quirks of each of our set-ups... some people need this (I do), but most don't. If you get a storage_engine error, you need this.